React/Views/RCTTVView.m

```diff
@@ -266,6 +296,55 @@ - (void)sendBlurNotification:(__unused UIFocusUpdateContext *)context
   object:@{@"eventType":@"blur",@"tag":self.reactTag}];
 }
 
+- (void)setNextFocusUp:(NSNumber *)nextFocusUp {
+  if (nextFocusUp == nil) return;
+  NSDictionary<NSNumber *, UIView *> *views = [_bridge.uiManager valueForKey:@"viewRegistry"];
+  RCTTVView *destination = (RCTTVView*)views[nextFocusUp];
```

Comment:
These two lines are the same in all four setter methods, maybe they should be a single helper method that takes an NSNumber and returns the corresponding view in the registry.

Reply:
I wonder if there is already a simple helper for this in RN somewhere?